examples/quart_example.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+import json
+from random import choice
+
+from quart import Quart
+
+from tortoise import Tortoise, fields
+from tortoise.models import Model
+
+STATUSES = ["New", "Old", "Gone"]
+app = Quart(__name__)
+
+
+class Users(Model):
+    id = fields.IntField(pk=True)
+    status = fields.CharField(20)
+
+    def __str__(self):
+        return "User {}: {}".format(self.id, self.status)
+
+
+class Workers(Model):
+    id = fields.IntField(pk=True)
+    status = fields.CharField(20)
+
+    def __str__(self):
+        return "Worker {}: {}".format(self.id, self.status)
+
+
+@app.before_serving
+async def init_orm():
+    await Tortoise.init(db_url="sqlite://:memory:", modules={"models": ["__main__"]})
+    await Tortoise.generate_schemas()
+
+
+@app.after_serving
+async def close_orm():
+    await Tortoise.close_connections()
+
+
+@app.route("/")
+async def list_all():
+    users, workers = await asyncio.gather(Users.all(), Workers.all())
+    return json.dumps(
```

Comment:
Quart has a `jsonify` function that should be used instead, as it sets the content-type header to json.

Reply:
Thanks, will change.